Sources/BraintreeThreeDSecure/Public/BraintreeThreeDSecure/BTThreeDSecureRequest.h

```diff
@@ -20,16 +20,30 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /**
- Error codes that describe errors that occur during 3D Secure.
+ 3D Secure version
```

Comment:
I think this was a copy-paste error at some point, so I fixed up this doc string.